src/deploy/osp_deployer/profiles/xsp.json

```diff
@@ -50,17 +63,17 @@
          }
       },
       {
-         "Advanced Settings":{
-            "rhsm_repos":"",
-            "use_internal_repo":"",
+         "Advanced Settings": {
+            "rhsm_repos": "",
+            "use_internal_repo": "",
             "deploy_overcloud_debug": {
                "valid_values": [
                   "true",
                   "false"
                ]
             },
-         "use_ipmi_driver":"",
-         "use_in_band_introspection":""
+            "use_ipmi_driver":"",
+            "use_in_band_introspection":""
```

Comment:
Probably want to change :"" to : "" on the 2 lines above for consistency in formatting.